### PR TITLE
updated 8th sept blog for office hours

### LINKED ISSUE #57: Add new blog for 8th september office hrs
Add new blog for 8th september office hrs: 08.09.2022

### PR DESCRIPTION
**Pull request by 26tanishabanik**
Signed-off-by: 26tanishabanik <26tanishabanik@gmail.com>
Fixes #57

### CODE REVIEW

File: blog/2022-09-8-office-hours.mdx [hunk 1]
```diff
@@ -44,7 +44,7 @@ that  when a contributor adds something to the examples, or to the open source s
 
 You can jump in here, the [KubeSlice Roadmap](https://github.com/orgs/kubeslice/projects/4/views/1), and feel free to take up any issue and again you can reach out on the  #kubeslice channel on the [Kubernetes slack](https://slack.k8s.io/) where we hang out, if you have questions about something or if you need any guidance and support. 
 Additionally, the Office Hours are intended 
-to be as interactive as possible and we’d love you to join us on our next one to be held on 8th September 2022. 
+to be as interactive as possible and we’d love you to join us on our next one to be held on 22nd September 2022. 
 To receive an invite for the next one in your inbox & stay updated with the latest goings-on in the project, [join our Google Group](https://groups.google.com/g/kubeslice-dev).
```
Comment (amolmote) [inline blog/2022-09-8-office-hours.mdx:48]: invite -> invitation , looks better here

File: blog/2022-09-8-office-hours.mdx [hunk 2]
```diff
@@ -9,7 +9,7 @@ hide_table_of_contents: false
 ---
 
 
-The [KubeSlice project](https://github.com/kubeslice) recently hosted the second episode of its Office Hours series on Thursday, 8th September, 2022
+The [KubeSlice project](https://github.com/kubeslice) recently hosted the third episode of its Office Hours series on Thursday, 8th September, 2022
```
Comment (amolmote) [inline blog/2022-09-8-office-hours.mdx:12]:
```suggestion
The [KubeSlice project](https://github.com/kubeslice) recently hosted the third episode of its Office Hours series on Thursday, 8th September 2022
```

Comment (amolmote) [reply blog/2022-09-8-office-hours.mdx:12]: no need to provide this comma

**Review by amolmote — COMMENTED** [review]
Looks good, i have suggested only one changes but that also not mandatory @26tanishabanik

File: blog/2022-09-8-office-hours.mdx [hunk 3]
```diff
@@ -9,7 +9,7 @@ hide_table_of_contents: false
...
 
 In this episode, [Eric Peterson](https://www.linkedin.com/in/eric-peterson-90073b2/), Vice President Of Engineering at [Avesha](https://avesha.io/) walked us through the road map, introduced us to multi-cluster, which is one of the basic issues, KubeSlice is attempting to address and demonstrated the connectivity to Kubernetes as well as Kubernetes' support for multiple tenants works.
```
Comment (divya-mohan0209) [inline blog/2022-09-8-office-hours.mdx:14]:
```suggestion
In this episode, [Eric Peterson](https://www.linkedin.com/in/eric-peterson-90073b2/), Vice President Of Engineering at [Avesha](https://avesha.io/) walked us through the road map and introduced us to the challenges of multi-cluster multi-tenancy in Kubernetes that KubeSlice is trying to address.
```

File: blog/2022-09-8-office-hours.mdx [hunk 4]
```diff
@@ -44,7 +44,7 @@ that  when a contributor adds something to the examples, or to the open source s
 
 You can jump in here, the [KubeSlice Roadmap](https://github.com/orgs/kubeslice/projects/4/views/1), and feel free to take up any issue and again you can reach out on the  #kubeslice channel on the [Kubernetes slack](https://slack.k8s.io/) where we hang out, if you have questions about something or if you need any guidance and support.
```
Comment (divya-mohan0209) [inline blog/2022-09-8-office-hours.mdx:45]:
```suggestion
He walked us through the various issues on [KubeSlice roadmap](https://github.com/orgs/kubeslice/projects/4/views/1) and explained the context behind the creation of each one. If there's anything in particular that you feel you could work on, feel free to assign it on your name. If you are unable to, please reach out to us on the  **#kubeslice** channel in the [Kubernetes slack](https://slack.k8s.io/). 
```

File: blog/2022-09-8-office-hours.mdx (hunk 1 above)
Comment (divya-mohan0209) [reply blog/2022-09-8-office-hours.mdx:48]: I disagree because you do not send out meeting invitations, you send out meeting invites.

**Review by divya-mohan0209 — COMMENTED** [review]
Initial set of changes

**Comment by divya-mohan0209** [comment]
Also, @26tanishabanik can we please wrap text while submitting so that reviewing is easier?

**Comment by divya-mohan0209** [comment]
@26tanishabanik Gentle reminder

**Comment by veenamj-avesha** [comment]
We don't have blogs on our OSS website now. They are moved to the marketing website. This blog describes an outdated event. 

So, I'm closing this issue.

**Comment by veenamj-avesha** [comment]
This is on an outdated events and we don't have blogs on our OSS website now.

**Pull request closed without merging**